CoreScriptsRoot/Modules/VR/Recenter.lua

```diff
@@ -72,6 +78,12 @@ function RecenterModule:SetVisible(visible)
 			isCountingDown = true
 			countdownPanel:SetVisible(true)
 
+			ContextActionService:BindCoreAction(cancelShortcutName, function(actionName, inputState, inputObj)
+				if inputState == Enum.UserInputState.Begin then
+					cancelCountdown()
+				end
+			end, false, Enum.KeyCode.ButtonB)
+
```

Comment:
Is it possible to spawn off the count down logic? I don't think waiting in a `SetVisible` function is great because it yields other threads for something that doesn't seem like a yielding call.